generated/nirfmxspecan/nirfmxspecan.proto

```diff
@@ -5889,7 +5907,6 @@ message MarkerCfgYLocationResponse {
 message MarkerFetchFunctionValueRequest {
   nidevice_grpc.Session instrument = 1;
   string selector_string = 2;
-  double timeout = 3;
```

Comment:
This is breaking too, and not related to the other ones. I added [a comment in the scrapigen review ](https://github.com/ni/grpc-device-scrapigen/pull/230/files#r1592921949) about what is causing this but I'm not sure if it's intended or not on that end.

Reply:
The issue what we are seeing now was fixed in the PR: #1049. This was a permanent fix as we had added the (hard-coded) desried function values in functions_addons.py of the respective personalities. 

But when I used scrapigen to generate the code this time, It is not able to pick up the data from the functions_addons.py file and instead giving the same issue which was reported previously. 

Was there any logical change done on the scrapigen tool? If it is not clear, then we might have to file a bug.

Reply:
Same comment as above i.e. Even if I correct this source breaking change by adding appropriate code in functions_addons.py file, I won't be able to generate the desired code.

Reply:
@ThangamV-NI , Rahul mentioned in the scrapigen changes that there was [this work item](https://dev.azure.com/ni/DevCentral/_workitems/edit/2679274/) for removing this parameter: 

So, is this just an expected breaking change for the API / proto file for SpecAn?

Reply:
@reckenro Yes, It is.

Reply:
@reckenro I have updated the PR with source-breaking and binary-breaking tags

Reply:
Alright, sounds good then. FYI @maxxboehme that we'll have a breaking change this release for SpecAn.

Reply:
I guess since for this API grpc-device has never worked having this change is fine.